invenio/values.yaml

```diff
@@ -127,6 +127,20 @@ persistence:
   size: 10G
   storage_class: ""
 
+mq_persistence:
+  enabled: true
+  name: "rabbitmq-volume"
+  access_mode: ReadWriteMany
+  size: 10G
+  storage_class: "cephfs-ssd-no-backup"
```

Comment:
This is a CERN value. As discussed, maybe make it `""`? (it should take the default)
We need then to override this in our CERN configs and use this value there.